### ISSUE TITLE
"test@example.com [any@thing@here]" passes RFCValidation

### ISSUE THREAD

**Issue by Nacoma** [egulias/EmailValidator#214]
```php
"test@example.com []";
"test@example.com [any@thing@here]";
"test@example.com [outlook:bad]";
```

These addresses are parsed into:

```php
[
    "local" => "test",
    "domain" => "example.com ]"
]
```
by the email parser and pass using the `RFCValidation` validator.

The `NoRFCWarningsValidation` validator does reject this.

**Comment by egulias** [comment]
Hi @Nacoma , you are right, this is a bug. Thanks for reporting.

**Comment by egulias** [comment]
See #217

**Comment by egulias** [comment]
Solved in 2.1.22
